core_backend/app/config.py

```diff
@@ -22,7 +22,9 @@
 # "model_name" in the proxy config.yaml.
 # "openai/..." is needed since the proxy presents a unified OpenAI-style API
 # for all of its endpoints.
-LITELLM_MODEL_EMBEDDING = os.environ.get("LITELLM_MODEL_EMBEDDING", "openai/embeddings")
+LITELLM_MODEL_EMBEDDING = os.environ.get(
+    "LITELLM_MODEL_EMBEDDING", "openai/local-embeddings"
```

Comment:
I don't think we want to use local-embeddings by default. Requires setting up an optional component! Let's change this back to embeddings? Also see my comment in the litellm config

Reply:
I added it for easy testing since the embeddings default size is 1536. I will put it back to the default value when we will fix the embeddings size on main.

Reply:
Not sure I understand -- can you elaborate on this?

Also, I don't think we want to swap embeddings with gte-large just yet. We should evaluate some other options and switch?

Reply:
Let's discuss this on slack first

Reply:
I put the embeddings size back to 1536